CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.08-alpha2
```

Comment:
Please remove the changelog entry. This is for releases (managed by me) and this will be part of a new version not an alpha release.

Reply:
Removed